lib/modules/@apostrophecms/busy/src/apos/components/TheApostropheBusy.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="apos-busy">
+    <div v-if="busy" class="busy"></div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'TheApostropheBusy',
+  data() {
+    return {
+      busy: false
+    }
+  },
+  mounted() {
+    apos.bus.$on('apos-busy', state => (this.busy = state.active));
```

Comment:
This is the global, blocking busy indicator right? So it should pay attention to `state.name` and ignore the event if it is not `busy`.